src/test/java/bio/terra/service/duos/DuosDaoTest.java

```diff
@@ -83,6 +92,10 @@ public void testInsertThenRetrieveFirecloudGroup() {
 
     DuosFirecloudGroupModel retrievedById = duosDao.retrieveFirecloudGroup(duosFirecloudGroupId);
     verifyRetrievedFirecloudGroupContents(retrievedById);
+
+    List<DuosFirecloudGroupModel> retrieved = duosDao.retrieveFirecloudGroups();
+    assertThat(retrieved.size(), equalTo(1));
```

Comment:
There are matchers for collection length
```suggestion
    assertThat(retrieved, hasSize(1));
```